governance/votes/Executive vote - May 18, 2022.md

```diff
@@ -0,0 +1,75 @@
+- 541.47 MKR will be distributed to the [Sustainable Ecosystem Scaling Core Unit (SES-001)](https://mips.makerdao.com/mips/details/MIP39c2SP10) as detailed below.
+
+If this executive proposal passes, the following **additions** will be made to the Maker Protocol:
+- A new `DssCure` module will be added and authorized.
+- `MCD_END` will be replaced.
+- The existing ESM contract will point to the new `END`.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### Core Unit MKR Vesting Streams
+
+As per their successful [MKR budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP17), a total of **541.47 MKR** will be distributed to the [Sustainable Ecosystem Scaling Core Unit (SES-001)](https://mips.makerdao.com/mips/details/MIP39c2SP10) at [0x87AcDD9208f73bFc9207e1f6F0fDE906bcA95cc6](http://etherscan.io/address/0x87AcDD9208f73bFc9207e1f6F0fDE906bcA95cc6) (SES-001's Auditor Wallet).
+
+### Addition of the `DssCure` Module and Related Modifications
+
+#### Context
+
+The [Maker Teleport](https://forum.makerdao.com/t/introducing-maker-wormhole/11550) initiative, [previously known as Maker Wormhole](https://forum.makerdao.com/t/maker-wormhole-new-name-maker-teleport/15115), will be bringing canonical DAI capability onto L2 domains. As part of this development it is necessary to introduce a new module called `DssCure` and update the existing `End` module to interact with it.
+
+#### What It Is
+
+In the event of an [Emergency Shutdown](https://docs.makerdao.com/smart-contract-modules/shutdown), the `DssCure` module ensures a fair distribution to DAI holders by calculating how much debt needs to be reduced where there is pre-minted, unused DAI sitting in the `Vat`. Discounting these pre-minted, out-of-circulation DAI is is necessary for the correct calculation of the outstanding DAI---otherwise, the artificial, increased debt resulting from factoring in the pre-minted DAI will result in DAI holders receiving less collateral than they are entitled to.
+
+`DssCure` sits between the `Vat` and `End` so that `End` receives the correct amount of debt.
+
+More generally, `DssCure` has the capability to add or remove sources from circulation.
+
+For more details, please read [this forum post by the Protocol Engineering Core Unit (PE-001)]().
+
+#### Additions and Modifications
```

Comment:
Would remove this title, and move this section above the 'For more details' sentence above. Make sure to include the "The following changes will take place if this executive proposal passes"